### PR TITLE
fix(npm): misc syntax highlighting

### PR DESCRIPTION
**Pull request by somerand0mcat**
## 🔧 What does this fix? 🔧
as said in the title
image
image
sorry if my PRs are a bit bearable to review

## 🗒 Checklist 🗒

- [X] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [X] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/npm/catppuccin.user.css [hunk 1]
```diff
@@ -135,6 +135,15 @@
...
     }
 
+    /* Syntax highlighting text */
+    span[style*="color: rgb(223, 80, 0);"] {
+      color: @peach !important;
+    }
+    
+    span[style*="color: rgb(99, 163, 92);"] {
+      color: @green !important;
+    }
```
Comment (uncenter) [inline styles/npm/catppuccin.user.css:145]: Why put it here? Pretty sure I already have a section somewhere for all other syntax highlighting. Be sure to follow the organization pattern of each userstyel you contribute to.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/npm/catppuccin.user.css (hunk 1 above)
Comment (somerand0mcat) [reply styles/npm/catppuccin.user.css:145]: i didnt know there was a section for syntax highlighting but i'll move it there anyway

**Review by uncenter — APPROVED** [review]

File: styles/npm/catppuccin.user.css (hunk 1 above)
Comment (somerand0mcat) [reply styles/npm/catppuccin.user.css]: im not going to be like last time when i only moved the comment

Comment (uncenter) [reply styles/npm/catppuccin.user.css]: A command f search for "syntax" would find you the section. Take the time to do things properly.

**Pull request merged**